### ISSUE TITLE
Cloud implementations

### ISSUE THREAD

**Issue by markfirmware** [blippy/ultimc#12]
This uses qemu's built-in vnc server. Try vnc 212.71.245.69 port 5990

Work: https://github.com/markfirmware/ultimc/tree/linode - wip - do not merge

Usage:

    wget https://github.com/markfirmware/ultimc/releases/download/v6.20181003/ultimc-v6.20181003.tgz
    tar zxf ultimc-v6.20181003.tgz
    cd release
    ./linode-install-and-run

**Comment by blippy** [comment]
Yes, I need to figure out what to do about that. Maybe it needs to go into its own framebuffer. For now, keep pressing return until the blackness disappears.
